azurepipelines/build/docker/adu-debian-arm32-build.yml

```diff
@@ -95,16 +95,20 @@ extends:
                 parameters:
                   targetOs: "debian11"
                   targetArch: "arm32"
+                  additionalDependencies: "--install-catch2"
 
-          - job: BuildAduAgent_debian10
-            displayName: "Build ADU Agent - Debian 10 (arm32)"
-            timeoutInMinutes: 120
-            cancelTimeoutInMinutes: 120
-            pool:
-              name: aduc_1es_client_pool
-              os: linux
-            steps:
-              - template: /azurepipelines/build/templates/adu-docker-build-steps.yml@self
-                parameters:
-                  targetOs: "debian10"
-                  targetArch: "arm32"
+        #
+        #   DISCONTINUED
```

Comment:
Perhaps just delete this -- the git history will have it in case need to resurrect it in the future.